configs/_base_/datasets/kitti_seg.py

```diff
@@ -0,0 +1,54 @@
+data_root = 'data/kitti-seg/'
+dataset_type = 'KittiSegDataset'
+
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+crop_size = (864, 256)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations'),
+    dict(type='Resize', img_scale=(1242, 375), ratio_range=(0.5, 2.0)),
+    dict(type='RandomCrop', crop_size=crop_size, cat_max_ratio=0.75),
+    dict(type='RandomFlip', prob=0.5),
+    dict(type='PhotoMetricDistortion'),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='Pad', size=crop_size, pad_val=0, seg_pad_val=255),
+    dict(type='DefaultFormatBundle'),
+    dict(type='Collect', keys=['img', 'gt_semantic_seg']),
+]
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='MultiScaleFlipAug',
+        img_scale=(1232, 368),
```

Comment:
The `img_scale` may set to be the largest scale.
https://github.com/open-mmlab/mmsegmentation/blob/46326f63ce411c794d237e986dd3924590d0e75e/docs/en/tutorials/config.md?plain=1#L131

Reply:
Reference :  https://github.com/NVIDIA/semantic-segmentation/blob/2f548ab30ab0d56e91de66a4dea4757a0c64e7e4/datasets/kitti.py#L222-L225